cmd/kpod/formats/formats.go

```diff
@@ -60,33 +65,34 @@ func (j JSONStructArray) Out() error {
 
 // Out method for Go templates
 func (t StdoutTemplateArray) Out() error {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
```

Comment:
No it doesn't I believe. This just aligns the fields automatically based on the width of the longest info printed for each column.